### DISCUSSION TITLE
Local Terminal and WSL Configuration Location

### DISCUSSION THREAD

**Discussion by alextibor** [Eugeny/tabby#8496] [Q&A]
Where can I configure the local terminal and WSL?

I'm using Windows 10.

I mistakenly cleared the Tabby settings and now I can't find the place to configure the local terminal (CMD, PowerShell) and WSL in the tabby. Can someone help me please?

In profiles you only have the option of Raw Socket, SSH, Serial and Telnet.

**Comment by Eugeny** [comment]
Check that the "local" plugin isn't disabled under Settings > Plugins

**Comment by alextibor** [comment]
@Eugeny Thanks Eugene!
